pkg/execution/execution_test.go

```diff
@@ -402,3 +405,33 @@ func TestExecutionInfo(t *testing.T) {
 		})
 	}
 }
+
+func TestAbortTest(t *testing.T) { //nolint: tparallel
+	t.Parallel()
+
+	rt := goja.New()
+	ctxPtr := new(context.Context)
+	*ctxPtr = common.WithRuntime(context.Background(), rt)
+	*ctxPtr = lib.WithState(*ctxPtr, &lib.State{})
+	m, ok := New().NewModuleInstancePerVU().(*Execution)
+	require.True(t, ok)
+	m.WithContext(func() context.Context { return *ctxPtr })
```

Comment:
Did you base the core PR on the WIP one of mine to use this ?

Because I am pretty sure the `common.Bind` should work just as fine :thinking:

Reply:
Not because of this, but I needed your PR for #2 which requires it, and this PR is based on that.

Reply:
This is now based on https://github.com/grafana/k6/pull/2108.